site/documentation/content/api/command-router/index.md

```diff
@@ -0,0 +1,174 @@
+---
+title: "Command Router API Specification"
+linkTitle: "Command Router API"
+weight: 425
+resources:
+  - src: preconditions.svg
+  - src: register_cmd_consumer.svg
+  - src: unregister_cmd_consumer.svg
+  - src: set_last_known_gateway_success.svg
+---
+
+The *Command Router API* is used by *Protocol Adapters* to configure the Command Router service component, enabling it to route command & control messages to the protocol adapters that the target devices are connected to.
+<!--more-->
+
+The Command Router API is defined by means of AMQP 1.0 message exchanges, i.e. a client needs to connect to Hono using an AMQP 1.0 client in order to invoke operations of the API as described in the following sections.
```

Comment:
needs to connect to the service component?

Reply:
Yes. That's something to be changed in some of the other API docs as well (credentials, device-registration, etc).